recipe/meta.yaml

```diff
@@ -30,12 +30,13 @@ requirements:
     - josepy >=1.13.0
     - parsedatetime >=2.4
     - pyrfc3339
-    - python
     - pytz >=2019.3
-    - pywin32 >=300  # [win]
+    - pywin32-on-windows
     - setuptools >=41.6.0
     - zope.component
     - zope.interface
+  run_constrained:
+    - pywin32 >=300
```

Comment:
pywin32-on-windows Always ships with the same version so this makes sure that the correct one is installed.